bandData.js

```diff
@@ -0,0 +1,409 @@
+const bandNames = ['Down Home Agita',
+  'Amperstellar',
+  'Azure Nation',
+  'Tonephile',
+  'Catastrophe',
+  'Chronic Caterwaulers',
+  'The Alligator Miseries',
+  'Eggcorn',
+  'Highfalutin',
+  'Lushpuppy',
+  'Nine Percent Pockets',
+  'Lackamusical',
+  'Majordemo',
+  'Bass Menagerie',
+  'Mnemonic Miasma',
+  'Murmuration',
+  'Onomatopie',
+  'PseudoSounds',
+  'Slapdash',
+  'Martini California',
+  'Staccato',
+  'Toothsome',
+  'Uproar To Ghost 128',
+  'Boss Beesleep Ave',
+  'The Dreamer Showgirls',
+  'Swallow Tail Galactic',
+  'Pedal 405',
+  'Bop Rainbows',
+  'Purple Harmony',
+  'Sleep January',
+  'Cookie Demagogue',
+  'Doom Bachelor',
+  'The Head Around',
+  'Onslaught Motel',
+  'Below Ground',
+  'Insurrectionaries',
+  'Tadpole',
+  'The River to Malta',
+  'Toxic Thrust',
+  'Grave Sonic',
+  'The Texas Between Radios',
+  'The Hang Ten Metals',
+  'Fiddle Over Saddle',
+  'Lucky Hammers',
+  'The Lineup',
+  'Beyond Hurricanes',
+  'The Dixie Cacophony',
+  'Mother Williams Bang',
+  'Satan Skirmish',
+  'The Desert Lady Shuffle',
+  'Coconut 81',
+  'California Harleys',
+  'Flame Jimmy Din',
+  'Air Above Commotion',
+  'Trauma Turbulence',
+  'Tropical Cops',
+  'Do Clang',
+  'The Roadside Blinks',
+  'Revolting Crocodiles',
+  'Diner Pioneers',
+  'The Impact Noises',
+  'The Aggression',
+  'Baby Blade',
+  'The Tonality Cats',
+  'Clatter Around Aires',
+  'Bloody Fin',
+  'Pandemonium Below',
+  'The Settler Snap Cannons',
+  'Evil Stingray',
+  'Velvet Ambush',
+  'Mustang Wednesday',
+  'Bikini Foot',
+  'Highway Onslaught',
+  'Continental Yelp',
+  'The Jangles',
+  'Zombie Drive-Thru',
+  'Ghost Fuss',
+  'Dice Outbreak 53',
+  'Lament Lounge',
+  'Claw Snake',
+  'The Mighty Blitzes',
+  'Roller Mandolin',
+  'Honeybee Elephants',
+  'Shouting Discord',
+  'Urban Malcontent',
+  'Zebra Punch',
+  'One-Eyed Roulette',
+  'Hot Chorus',
+  'Hoo-Ha 77',
+  'Violent Ray',
+  'Chainmail Spiders',
+  'The Light Beyond Opal Valley',
+  'Gumdrop Spaceship',
+  'Lucid Apostate',
+  'The Watusi Encroachment',
+  'Secret Progeny',
+  'After The Missles',
+  'Finger Brawl',
+  'The Fearsome Onrush',
+  'Butterfly Blvd.',
+];
+
```

Comment:
As mentioned in your PR, don't forget to remove this bandName array in lieu of your Faker data :)